CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### PaymentSheet
 * [FIXED][10327](https://github.com/stripe/stripe-android/pull/10327) Improve accessibility for postal code.
+* [Changed][10382](https://github.com/stripe/stripe-android/pull/10382) Hide Google Pay price label when amount is 0.
```

Comment:
We should reword this to be for setup intents. 

`Don't show price as $0.00 for setup intents.`